db/migrate/20171124095557_create_join_table_events_users.rb

```diff
@@ -0,0 +1,8 @@
+class CreateJoinTableEventsUsers < ActiveRecord::Migration[5.1]
+  def change
+    create_join_table :events, :users do |t|
+      # t.index [:event_id, :user_id]
+      # t.index [:user_id, :event_id]
```

Comment:
indices could be used for faster querying of data.

Reply:
I dont think that its necessary